pkg/oidc/oidc.go

```diff
@@ -60,6 +63,7 @@ func (a *Authenticator) succeeded(w http.ResponseWriter, claims Claims) {
 }
 
 func (a *Authenticator) failed(w http.ResponseWriter, err error) {
+	log.WithError(err).Error("failed to authenticate")
```

Comment:
Is this an error or an expected case?

Reply:
It's an error, basically the OIDC dance fails.

Reply:
If you think OIDC dance failure is not an error, what log level is appropriate?

Reply:
I think if there is an *error* (something unexpected happened) in the OIDC dance, it is an error. If the auth failed, that is not an error, it is just information.